core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -551,7 +546,9 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
     ngOnDestroy(): void {
         this.destroy$.next(true);
         this.destroy$.complete();
-        this.uveStore.setIsClientReady(false);
+        if (this.uveStore.isTraditionalPage()) {
+            this.uveStore.setIsClientReady(true);
+        }
```

Comment:
Why headless doesn't need this?

Reply:
Because headless triggers the change when it is ready. I think this is a change from the merge

Reply:
@zJaaal I don't think it worth because we are going to get ride of that property when doing the UVE/SDK improvements